src/api/net.rs

```diff
@@ -36,6 +36,14 @@ impl<T: Transport> Net<T> {
         CallFuture::new(self.transport.execute("net_peerCount", vec![]))
     }
 
+    /// Returns list of connected nodes
+    pub fn peer_list(&self, _n: NodeType) -> CallFuture<PeerType, T::Out>{
```

Comment:
use `_` prefix only for unused parameters

Reply:
While I don't fully agree with placing it in the `Net` namespace, I don't have a good alternative to this. Let's leave it here then :)

Reply:
Fixed param prefix. Wasn't sure about putting it in it's own namespace. Saw that peer_count was under Net. Will leave it under Net until told otherwise.